src/main/java/com/car/app/App.java

```diff
@@ -0,0 +1,62 @@
+package com.car.app;
+
+import java.util.Date;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+//import org.slf4j.Logger;
+//import org.slf4j.LoggerFactory;
+
+import com.car.app.model.Car;
+
+/**
+ * @author selhanaf
+ *
+ * this is the main class used to insert/add new record to the data base
+ */
+public class App {
```

Comment:
This class is not necessary, since you're using the property hibernate.hbm2ddl.import_files to load cars.